File: C++/Edwin Villarraga/matematica/EulerTotient.cpp
#include<bits/stdc++.h>
#include<cstdlib>
using namespace std;
int gcd(int a,int b){
    if(a==0){
        return b;
    }
    return gcd(b%a,a);
}
int phi(int n){
    int result=1;
    for(int i=2;i<n;i++ ){
        if(gcd(i,n)==1){
            result++;
        }
    }
    return result;
}
int main() {
    for(int i=1;i<=10;i++){
        printf("el phi (%d) = %d\n",i,phi(i));
    }
}

File: PY/Bryann Valderrama/Matemática/eulerTotient.py
from sys import stdout
wr = stdout.write


def gcd(a, b):
    if a == 0:
        return b
    return gcd(b % a, a)


def phi(n):
    result = 1
    for i in range(2, n):
        if gcd(i, n) == 1:
            result += 1
    return result


for i in range(1, 11):
    wr(f'Phi ({i}) = {phi(i)}\n')
